rome/src/test/java/com/rometools/rome/io/XmLReaderHttpHeaderTest.java

```diff
@@ -1,45 +0,0 @@
-package com.rometools.rome.io;
-
-import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
-import static com.github.tomakehurst.wiremock.client.WireMock.*;
-
-import com.github.tomakehurst.wiremock.junit.WireMockRule;
-
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-
-import java.io.IOException;
-import java.net.URL;
-import java.util.HashMap;
-import java.util.Map;
-
-public class XmLReaderHttpHeaderTest {
```

Comment:
Wiremock is not used anymore and can be removed from pom.xml in rome-core and rome-parent

Reply:
OK, nice!

Reply:
I just thought about it once again. We should leave the test as-is until we finally remove the URL support in XmlReader

Reply:
Finally I restored the file.